core/cautils/versioncheck.go

```diff
@@ -4,13 +4,15 @@ import (
 	"context"
```

Comment:
it would make sense to move this to maybe armosec/utils-go since I want to call that from operator too